tests/tasks/setup/eks/awscli-fargate.yaml

```diff
@@ -60,28 +64,31 @@ spec:
           echo "$(params.profile-name) is "CREATING" at $(date)"
           sleep 2
       done
-  - name: validate-nodes
-    image: alpine/k8s:1.22.6
+  - name: write-kubeconfig
+    image: alpine/k8s:1.23.7
     script: |
-      echo "validate fargate nodes"
       ENDPOINT_FLAG=""
       if [ -n "$(params.endpoint)" ]; then
         ENDPOINT_FLAG="--endpoint $(params.endpoint)"
       fi
-      aws eks $ENDPOINT_FLAG update-kubeconfig --name $(params.cluster-name) --region $(params.region)
-      kubectl patch deployment coredns -n kube-system --type json -p='[{"op": "remove", "path": "/spec/template/metadata/annotations/eks.amazonaws.com~1compute-type"}]' || true
-      kubectl version
-      kubectl config current-context
-      kubectl get pods -n kube-system -o wide
-      kubectl get ns
-      kubectl get nodes
-      kubectl scale deployment/coredns --replicas=$(params.desired-nodes) -n kube-system
+      aws eks $ENDPOINT_FLAG update-kubeconfig --name $(params.cluster-name) --region $(params.region) --kubeconfig $(workspaces.config.path)/kubeconfig
+  - name: validate-nodes
+    image: alpine/k8s:1.23.7
+    script: |
+      echo "validate fargate nodes"
+      kubectl --kubeconfig $(workspaces.config.path)/kubeconfig patch deployment coredns -n kube-system --type json -p='[{"op": "remove", "path": "/spec/template/metadata/annotations/eks.amazonaws.com~1compute-type"}]' || true
```

Comment:
same comment as earlier is applicable to this task too.

Reply:
removed